pkg/observation_test.go

```diff
@@ -1,4 +1,4 @@
-package ocr2keepers
+package ocr2keepers_test
```

Comment:
Why use a different package name for the tests here?

Reply:
There was an import cycle and it was easier to rename the test package to side step it rather than move around implementation packages 👍